apps/services/search-indexer/src/migrate.ts

```diff
@@ -650,4 +688,6 @@ async function migrateBootstrap() {
 
 migrateBootstrap().catch((error) => {
   logger.error('ERROR: ', error)
+  // allow critical errors in migration to take down the process
+  throw error
```

Comment:
I would skip this throw for now. It is very useful to allow the migration container (running as the initContainer for the indexer) to finish successfully for now. Then we can possibly jump into the container to see what is really going on.

The main reason is that this migration process is quite complex. It needs to contact many different services and all have to have the correct status to allow this migration to continue.

But after this initial migration is successful, then we really should add this throw in here.

Reply:
Will remove